### ISSUE TITLE
[question] Garnish by default?

### ISSUE THREAD

**Issue by mattdesl** [mattdesl/budo#95]
Since version 5, budo has become a more general purpose and high-level tool for quick prototyping. The un-opinionated bits have been mostly split into their own modules ([watchify-middleware](https://www.npmjs.com/package/watchify-middleware), [simple-html-index](https://www.npmjs.com/package/simple-html-index), [get-ports](https://www.npmjs.com/package/get-ports), etc).

So maybe it's finally time `garnish` gets bundled in by default with budo. :smile: Users seeking ndjson can just do this:

```sh
budo index.js --ndjson | my-custom-printer
```

This is a breaking change and will probably cause some confusion the first time somebody uses this new version and tries to pipe to garnish.

**Comment by yoshuawuyts** [comment]
I agree with your reasoning, think it's ok to do

**Comment by wbinnssmith** [comment]
As much as the purist in me might disagree, I think this is the way to go. Nearly every time I've used budo I've piped its output through garnish. For beginners the ndjson might be intimidating and piping stdout through garnish might seem unnecessary. In general I think its leaking the (quite nice) modularity and composability of itself. Plus the `--ndjson` provides a nice fallback for those who prefer it or need to pipe it elsewhere.

That being said, one can configure `budo | garnish` in npm scripts and just use `npm run`.
